src/renderer/components/watch-video-comments/watch-video-comments.vue

```diff
@@ -97,6 +109,8 @@
             {{ comment.numReplies }}
             <span v-if="comment.numReplies === 1">{{ $t("Comments.Reply").toLowerCase() }}</span>
             <span v-else>{{ $t("Comments.Replies").toLowerCase() }}</span>
+            <span v-if="comment.hasOwnerReplied && !comment.showReplies"> {{ $t("Comments.From $").toLowerCase().replace("$", channelName) }}</span>
```

Comment:
It feels like this is like "passing variable to translation" (using `replace` is fine)
https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations

So I suggest having named placeholders
```suggestion
            <span v-if="comment.hasOwnerReplied && !comment.showReplies"> {{ $t("Comments.From $channelName").toLowerCase().replace("$channelName", channelName) }}</span>
```

Reply:
That makes a lot of sense to me but it also only uses the "$" in locales for other key values so I think a PR should probably be made to update it sometime in the future

Reply:
Sure, I will make that PR